### ISSUE TITLE
[web]: <i'm having this error with the flutter version for web each time i try to test it. But it works fine on mobile>

### ISSUE THREAD

**Issue by catenin95** [flutter/flutter#63071]
Everything works fine and App is already on the play store but as i said when i try to test for a web version it produces this error just for this particular App.

I Please will want to know what exactly is wrong

lib/main.dart:15:8: Error: Error when reading 'lib/generated_plugin_registrant.dart': The system cannot find the file specified.

import 'package:[NameOfApp goes here]/generated_plugin_registrant.dart';
       ^
lib/main.dart:16:8: Error: Error when reading 'lib/generated_plugin_registrant.dart': The system cannot find the file specified.de the command you're running, flutter_web version, browser version

**Comment by pedromassangocode** [comment]
Hi @catenin95 
Can you please provide your `flutter doctor -v`, your `flutter run --verbose`.
Thank you

**Comment by catenin95** [comment]
> Hi @catenin95
> Can you please provide your `flutter doctor -v`, your `flutter run --verbose`.
> Thank you

**Comment by catenin95** [comment]
> Hi @catenin95
> Can you please provide your `flutter doctor -v`, your `flutter run --verbose`.
> Thank you

I did.. bur it produced no error... Passed all checks right

**Comment by catenin95** [comment]
I was able to solve it using the Flutter SDk Command Line

I solved it by using the cd  command to navigate to where all my Flutter projects where  eg.

C:/Users/cd applications/ cd flutter projects/ cd ... Until I got to the exact flutter application folder that was giving me this issue.

Then using the command  "flutter create ."

It regenerated the folder for the web and all missing files that gave the errors were fixed.

**Comment by catenin95** [comment]
> Hi @catenin95
> Can you please provide your `flutter doctor -v`, your `flutter run --verbose`.
> Thank you

I did.. but it produced no error... It Passed all checks right

**Comment by catenin95** [comment]
I was able to solve it using the Flutter SDk Command Line

I solved it by using the cd  command to navigate to where all my Flutter projects where  eg.

C:/Users/cd applications/ cd flutter projects/ cd ... Until I got to the exact flutter application folder that was giving me this issue.

Then using the command  "flutter create ."

It regenerated the folder for the web and all missing files that gave the errors were fixed.
